DOCS/license-inclusion-principles.md

```diff
@@ -7,42 +7,41 @@ Determining whether a candidate license should be included on the SPDX License L
 * The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.
 * All OSI-approved licenses will be included on the SPDX License List.
   * This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also at that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.
-* Binary-only software licenses or licenses that do not provide for the availability of the source code will not be included on the SPDX License List.
+* Software licenses that apply only to executables and do not provide for the availability of the source code will not be included on the SPDX License List.
```

Comment:
```suggestion
* Software licenses that only license executables, and don't apply to source code, will not be included on the SPDX License List.
```

Reply:
so, executables instead of binaries is the key change here... I don't have a strong opinion on that one. 
@zvr @goneall @kestewart - thoughts?

Reply:
I don't have a strong opinion, but lean in favor of changing to "executables" as @kemitchell suggested. I think that's closer to capturing the intent of what we're trying to say, since not all programming languages' "executables" would be "binary-only" in a literal sense.

Reply:
change to executables seems to have favor. but I still think we need to keep "do not provide for availability of source code" as that's more specific than "don't apply to source code"

Reply:
Folks would say permissive licenses "don't provide for availability of source code".

Reply:
that was brought up before and I think this was the language that came out of that conversation - as in the source code is available (not requires availability downstream)

@swinslow just merged! you should have joined call, which just ended!! :)

Reply:
"only applies to executables" is also clarifying - no one would say that about MIT

Reply:
I can't do long standing calls during business hours. I need that time for client work and client calls.